wdl2cwl/tests/test_cwl.py

```diff
@@ -10,9 +10,18 @@ def get_file(path: str) -> str:
 
 def testCollectQualityYieldMetrics() -> None:
     """Test a single WDL to CWL conversion."""
-    abc = wdl.main([get_file("wdl_files/CollectQualityYieldMetrics.wdl")])
+    convertedStr = wdl.main([get_file("wdl_files/CollectQualityYieldMetrics.wdl")])
     testStr = ""
     with open(get_file("cwl_files/CollectQualityYieldMetrics.cwl")) as file:
         testStr = file.read()
 
-    assert abc == testStr
+    assert convertedStr == testStr
+
+def testSmoove() -> None:
+    """Test a single WDL to CWL conversion."""
+    convertedStr = wdl.main([get_file("wdl_files/smoove.wdl")])
+    testStr = ""
+    with open(get_file("cwl_files/smoove.cwl")) as file:
```

Comment:
Consider using https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions so you don't have to keep duplicating this test
```suggestion
@pytest.mark.parametrize("wdl_path,cwl_path",
    [("wdl_files/smoove.wdl", "cwl_files/smoove.cwl"),
     ("wdl_files/CollectQualityYieldMetrics.wdl", "cwl_files/CollectQualityYieldMetrics.cwl")])
def test_wdls(wdl_path, cwl_path: str) -> None:
    """Test WDL to CWL conversion."""
    convertedStr = wdl.main([get_file(wdl_path)])
    testStr = ""
    with open(get_file(cwl_path)) as file:
```

Reply:
probably needs an `import pytest` as well :-)